### PR TITLE
chore: Update dependencies

### PR DESCRIPTION
**Pull request by castastrophe**
Adding pfe-sass as a devDependency of any component using pfe-sass to compile.  This is important because when lerna looks at the dependency tree, if a component doesn't list it, it can lead to a race-condition where globbing is in process when another component's sass looks for it.

### Testing instructions

- `npm run build`

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Repository compiles and tests pass.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-codeblock/src/pfe-codeblock.scss [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
-@import "../../../node_modules/prismjs/plugins/line-numbers/prism-line-numbers";
 
+@import "pfe-codeblock-material-light";
+@import "pfe-codeblock-material-dark";
 
-$LOCAL: codeblock;
-
-// This variable is global so that helper functions can reference it
-$LOCAL-VARIABLES: ();
+@import "prismjs/plugins/line-numbers/prism-line-numbers";
```
Comment (castastrophe) [inline elements/pfe-codeblock/src/pfe-codeblock.scss:6]: Removed some of the unused boilerplate from here and simplified the imports -- sass includes the node_modules import path so no need to explicitly outline that.

**Comment by kylebuch8** [comment]
@castastrophe Can you create an issue for the PR and link it?

**Comment by castastrophe** [comment]
@kylebuch8 Done.  Are we requiring issues for every chore we work on?

**Comment by castastrophe** [comment]
https://github.com/patternfly/patternfly-elements/issues/1394

**Comment by castastrophe** [comment]
@kylebuch8 Where are we with this?  The issue has been added.  Wondering when we'll have time to get this update in?  It will help us bring back parallelization on compile.

**Review by kylebuch8 — APPROVED** [review]
Light Gone. Tonight's Murk.

**Pull request merged**